staging/src/k8s.io/client-go/rest/transport.go

```diff
@@ -94,7 +94,11 @@ func (c *Config) TransportConfig() (*transport.Config, error) {
 	}
 
 	if c.ExecProvider != nil {
-		provider, err := exec.GetAuthenticator(c.ExecProvider)
```

Comment:
would it make more sense to change this to `exec.GetAuthenticator(c)` and move ConfigToExecCluster to the `k8s.io/client-go/plugin/pkg/client/auth/exec` package which already has visibility to `k8s.io/client-go/tools/clientcmd/api` and `k8s.io/client-go/pkg/apis/clientauthentication`?

Reply:
hopefully this is resolved by https://github.com/kubernetes/kubernetes/pull/95489#discussion_r513892294